src/app/utils.ts

```diff
@@ -37,7 +35,8 @@ export const actionHook = <C extends AnyActionCreator>(actionCreator: C, body: A
           catchError(e);
         }
       }
-      return result;
+
+      return next(action);
```

Comment:
just curious - is this change changing any behavior - it shouldn't right?
However, maybe it was written this way to make sure that `matches(action)` or `boundHook(action)` does not cause any side effects?

Reply:
The reason why I moved it is so that the action hooks get the access to store that was not yet updated (as long as they are sync which is why i removed async from `updateHighlight` for example).

from redux docs: `The last middleware in the chain will receive the real store's dispatch method as the next parameter`
`The store's reducing function will be called ... synchronously. Its return value will be considered the next state. It will be returned from getState() from now on`

I will double check if any of the existing hooks don't depend on the opposite thing happening

Reply:
Turns out the pageView GA event depends on it, will need to rewrite the logic a bit.

Reply:
Nice catch, please let me know when you finish fixing this so I can review